example/ios/Flutter/ephemeral/flutter_lldb_helper.py

```
@@ -0,0 +1,32 @@
+#
+# Generated file, do not edit.
+#
+
+import lldb
+
+def handle_new_rx_page(frame: lldb.SBFrame, bp_loc, extra_args, intern_dict):
```

Ruff (0.14.5)

Unused function argument: `bp_loc`

(ARG001)